src/components/TextListItem.tsx

```diff
@@ -32,135 +34,192 @@ export type ItemData = {
+        badge={badge}
+        onPress={() => (onPress ? onPress(navigation) : navigate())}
+        disabled={!navigation}
+        delayPressIn={0}
+        Component={Touchable}
+        accessibilityLabel={`(${title}) ${consts.a11yLabel.button}`}
+      >
+        {leftIcon ||
+          (leftImage && !!picture?.url ? (
+            <Image
+              source={{ uri: picture.url }}
+              childrenContainerStyle={[
+                styles.smallImage,
+                imageStyle,
+                withCard && styles.withBigCardStyle
+              ]}
+              borderRadius={withCard ? normalize(8) : undefined}
+              containerStyle={[styles.smallImageContainer, imageContainerStyle]}
+            />
+          ) : undefined)}
+
+        {withCard ? (
+          <ListItem.Content>
+            {!!overtitle && (
+              <HeadlineText smallest uppercase style={styles.overtitleMarginBottom}>
+                {trimNewLines(overtitle)}
+              </HeadlineText>
+            )}
+            {noSubtitle || !subtitle ? undefined : titleText}
+            {noSubtitle || !subtitle ? (
+              titleText
+            ) : (
+              <RegularText small style={styles.subtitle}>
+                {subtitle}
+              </RegularText>
+            )}
+          </ListItem.Content>
+        ) : (
+          <ListItem.Content style={listItemStyle}>
+            {!noOvertitle && !!overtitle && (
+              <HeadlineText smallest uppercase style={styles.overtitleMarginBottom}>
+                {trimNewLines(overtitle)}
+              </HeadlineText>
+            )}
+            {noSubtitle || !subtitle ? undefined : titleText}
+            {noSubtitle || !subtitle ? (
+              titleText
+            ) : (
+              <RegularText small style={styles.subtitle}>
+                {subtitle}
+              </RegularText>
+            )}
+            {!!status && <RegularText>{status}</RegularText>}
+          </ListItem.Content>
+        )}
+
+        {rightIcon ||
+          (rightImage && !!picture?.url ? (
+            <Image
+              source={{ uri: picture.url }}
+              childrenContainerStyle={[styles.smallImage, withCard && styles.withBigCardStyle]}
+              borderRadius={withCard ? normalize(8) : undefined}
+              containerStyle={styles.smallImageContainer}
+            />
+          ) : undefined)}
+
+        {!!count && <BoldText>{count}</BoldText>}
+
+        {!listsWithoutArrows && !!navigation && !withCard && (
+          <Icon.ArrowRight color={colors.darkText} size={normalize(18)} />
+        )}
+      </ListItem>
+    );
+  }
+);
 /* eslint-enable complexity */
 
 const styles = StyleSheet.create({
   container: {
     backgroundColor: colors.transparent,
-    paddingVertical: normalize(12)
+    paddingHorizontal: 0,
+    paddingVertical: normalize(16)
+  },
+  overtitleMarginBottom: {
+    marginBottom: normalize(4)
   },
   smallImage: {
     height: normalize(72),
```

Comment:
ListItem's view seems to be missing padding on the left and right.

|||
|--|--|
![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 03 59](https://github.com/user-attachments/assets/cbfff534-3240-4558-ae43-60bfddb5156a)|![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 09 08](https://github.com/user-attachments/assets/af934bc1-72de-47ab-9a57-ea19009f705b)

Reply:
will be fixed

|||
|---|---|
|<img width="379" alt="image" src="https://github.com/user-attachments/assets/be1a390b-2669-4486-999b-a15479da259f">|<img width="383" alt="image" src="https://github.com/user-attachments/assets/7154be5e-4c34-4155-8d72-56b93923d362">|

Reply:
there could be even more places with flat lists, where the padding is missing now. @d4v3d4vE @ardasnturk please click around those places, where flat lists are used to find out.